build.gradle

```diff
@@ -257,6 +276,9 @@ task sourcesJar(type: Jar) {
  * This specifies what artifacts will be built and uploaded when performing a maven upload.
  */
 artifacts {
+    // Make sure the barclayJar is listed first to prevent it from being chosen as the one to be
+    // installed under the snapshot name in the local maven repository when doing "./gradlew install".
+    archives barclayJar
```

Comment:
I think we want to upload a non-shadow jar as the barclay jar we archive so that gatk can depend on it.  We probably need to add another jar task to make that.